.github/workflows/rat.yaml

```
@@ -42,12 +42,12 @@ jobs:
 
       - name: Download Apache RAT
         run: |
-          curl -L -O https://downloads.apache.org/creadur/apache-rat-0.16.1/apache-rat-0.16.1-bin.tar.gz
+          curl -L -O https://archive.apache.org/dist/creadur/apache-rat-0.16.1/apache-rat-0.16.1-bin.tar.gz
```

```suggestion
          curl -L -O https://downloads.apache.org/creadur/apache-rat-0.17/apache-rat-0.17-bin.tar.gz
```
I think, as rat had a new release recently.

---

Updated.